src/rules/sort-collections.ts

```diff
@@ -29,14 +29,38 @@ export const rule = createRule<Options>({
 					toSort.includes(key.value)
 				) {
 					const currentOrder = collection.properties;
-					const desiredOrder = currentOrder
-						.slice()
-						.sort((a, b) =>
-							(a.key as AST.JSONStringLiteral).value >
-							(b.key as AST.JSONStringLiteral).value
-								? 1
-								: -1,
-						);
+					const desiredOrder = currentOrder.slice().sort((a, b) => {
+						let aKey = (a.key as AST.JSONStringLiteral).value;
+						let bKey = (b.key as AST.JSONStringLiteral).value;
+
+						if (key.value !== "scripts") {
+							return aKey > bKey ? 1 : -1;
+						} else {
+							let modifier = 0;
```

Comment:
I'm not sold on how I did this - am curious to know your thoughts

Reply:
Having just one `let modifier` is clever! It seems reasonable if we assume `pre` and `post` always cause these modifiers. But, what happens if the script name just coincidentally happens to have `pre` or `post` and doesn't match other stuff?

```json
{
  "scripts": {
    "lint": "...",
    "poster": "..."
  }
}
```

I would think `"poster"` should come after `"lint"`. Could you add some test coverage for at least the following & make sure they work?

* `"pre"` without a non-prefixed script
* `"post"` without a non-prefixed script
* `"pre" and `"post"` without a non-prefixed script